Assets/Scripts/GameController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class GameController : MonoBehaviour {
+
+  public GUIText GameOverText;
```

Comment:
[ADVICE]
Rather than using Legacy UI Component such as `GUI Text`, using UI System is better.
https://docs.unity3d.com/Manual/UISystem.html